Src/ReleaseHistory.md

```diff
@@ -16,6 +16,10 @@
 - UER => eliminate unhandled exceptions in rules
 - UEE => eliminate unhandled exceptions in engine
 
+## Unreleased
+
+- RRR: Moving validators to separate folders and updating namespaces accordingly. [#610](https://github.com/microsoft/sarif-pattern-matcher/pull/610)
```

Comment:
>Moving [](http://example.com/codeflow?start=7&length=6)

Q: does this change impact runtime behaviors of our product?
A: No.

So why did you add a release note? Please remove.

Remember this question: does this change impact runtime behaviors of our product. You really need to understand this concept in order to maintain this file properly, it is intended to help external people who might be impacted by our changes.

It is not a general log of code changes. You might have been thinking our validators are a public API: they are not.

Reply:
Since I'm moving back the namespace, it is reasonable to remove this.

But, the validators API are all public. You, as a user, would be able to instantiate the AdoPatValidator and call `IsValidStatic`.
[https://github.com/microsoft/sarif-pattern-matcher/blob/1cb648695e1f3ea04e4ea9732f3e6939b19a5b0f/Src/Plugins/Security/SEC101_102.AdoPatValidator.cs#L12](https://github.com/microsoft/sarif-pattern-matcher/blob/1cb648695e1f3ea04e4ea9732f3e6939b19a5b0f/Src/Plugins/Security/SEC101_102.AdoPatValidator.cs#L12)

[https://github.com/microsoft/sarif-pattern-matcher/blob/1cb648695e1f3ea04e4ea9732f3e6939b19a5b0f/Src/Sarif.PatternMatcher.Sdk/StaticValidatorBase.cs#L12](https://github.com/microsoft/sarif-pattern-matcher/blob/1cb648695e1f3ea04e4ea9732f3e6939b19a5b0f/Src/Sarif.PatternMatcher.Sdk/StaticValidatorBase.cs#L12)

Reply:
Removed the line.